software/pelicanpanel.yml

```diff
@@ -0,0 +1,11 @@
+name: "Pelican Panel"
+website_url: "https://pelican.dev/"
+source_code_url: "https://github.com/pelican-dev/panel"
+description: "Web application for easy management of game servers, offering a user-friendly interface for deploying, configuring, and managing servers, server monitoring tools, and extensive customization options."
```

Comment:
It should probably be mentioned that it is a fork.
```suggestion
description: "Web application for easy management of game servers, offering a user-friendly interface for deploying, configuring, and managing servers, server monitoring tools, and extensive customization options (fork of Pterodactyl)."
```

Reply:
thanks for catching that 👍🏻